docs/DeveloperGuide.adoc

```diff
@@ -1286,3 +1286,57 @@ _{ more test cases ... }_
 
 _{ more test cases ... }_
 
+[appendix]
+== Effort
+
+In this section, we highlight the amount of effort taken for us to develop ExpenseLa, the challenges faced in this development and our eventual achievement.
+
+=== Effort by team
+
+Our application, ExpenseLa, is considerably different from what Address Book 3 (AB3) had implemented. We wanted to take some address book features
+that were available in AB3, but wanted to build an expense tracker for our project. As such, to build ExpenseLa, we took and modified some features previously
+available in AB3, but had to create many of our own models and implementations for the new features.
+
+In short, AB3 stores details related to multiple people known to the user. ExpenseLa stores data related to multiple transactions by
+the user and so much more. ExpenseLa uses that data to track the monthly expenditure, income as well as budget, analyses their expenditure
+by graphing it out, and provide many other features like filtering transactions as well as new ways of adding transactions to make the application
```

Comment:
Don't forget to include monthly data as well, which generates insights on the user's monthly expenditure

Reply:
Hmm this part is about challenges faced, so unless there's a specific challenge faced about this feature I don't think it's necessary to talk about it, as there is a seperate section for the implementation of the new feature Monthly Data